db_assessment/api.py

```diff
@@ -11,6 +11,8 @@
 @app.route("/api/loadAssesment", methods=["POST"])
```

Comment:
One thing I forgot to check before is that all files that support comments need to have the following comment in the very beginning of it per Google's policy. 

Can you please add it to all new files you created? Thanks ;-) 

# Copyright 2020 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     https://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

Reply:
Done!